src/share/wizard-steps.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import React, {PropTypes as t} from 'react';
+import './wizard-steps.css';
+
+type PropsType = {steps: Array<string>, activeIndex: number};
+
+const WizardSteps = ({steps, activeIndex}: PropsType) => (
+  <div className="wizard-steps">
+    {steps.map((s, i) => (
+      <div className="step" key={i}>
```

Comment:
I'm surprised ESLint didn't complain about using an index for a key. You could use `s` instead of `i` since there shouldn't be duplicate wizard steps.